gems/smithy/lib/smithy/templates/client/client.erb

```diff
@@ -133,14 +133,13 @@ module <%= module_name %>
         <%= protocols %>
       end
 
-
       # @api private
-      def identity_providers(context)
```

Comment:
I think we'll be able to just delete this method entirely because we can look at config.auth_scheme to get the providers to call.